test/shared/commonFunctionsForTests.sh

```diff
@@ -602,3 +603,95 @@ function score_parallelization_test() {
     diff "$rstFile1" "$rstFile2"
     return $?
 }
+
+
+function print_bootstrap_info_message() {
+    #========================================================================
+    # Prints a message to indicate if species bootstrapping is enabled.
+    #
+    # 1st argument: Enable ("yes") or disable ("no") bootstrapping
+    #========================================================================
+    if [[ "x${1}" == "xyes" ]]; then
+        echo ""
+        echo "%%%%% Species not found in the restart file will   %%%%%"
+        echo "%%%%% be bootstrapped (i.e. set to default values) %%%%%"
+    else
+        echo ""
+        echo "%%%%% Integration tests will fail unless all %%%%%"
+        echo "%%%%% species are found in the restart file! %%%%%"
+    fi
+}
+
+
+function gcc_enable_or_disable_bootstrap() {
+    #========================================================================
+    # Edits HEMCO_Config.rc files to enable or disable "bootstrapping",
+    # which is setting missing restart file variables to default values.
+    #
+    # 1st argument: Enable ("yes") or disable ("no") bootstrapping
+    # 2nd argument: Directory containing integration test rundirs
+    #========================================================================
+    bootStrap="${1}"
+    rundirsDir="${2}"
+
+    print_bootstrap_info_message "${bootStrap}"
+
+    # Loop over rundirs
+    for runDir in ${rundirsDir}/*; do
+
+        # Do the following if for only valid GEOS-Chem run dirs
+        expr=$(is_valid_rundir "${runDir}")
+        if [[ "x${expr}" == "xTRUE" ]]; then
+
+            # Specify path HEMCO_Config.rc
+            hcoCfg="${runDir}/HEMCO_Config.rc"
+
+            if [[ "x${bootStrap}" == "xyes" ]]; then
+                # Set missing restart file variables to defaults
+                sed_ie "s/EFYO/CYS/"             "${hcoCfg}"
```

Comment:
Is there any change HEMCO_Config.rc might have EFY0, EFY, etc in the future? It would probably be safer to include the variable name in the search criteria in case we add anything in the future with these settings.

Reply:
@lizziel: I can fix this as well.

Reply:
BTW, I think this will work.  Testing it now:
```bash
            if [[ "x${bootStrap}" == "xyes" ]]; then
                # Set missing restart file variables to defaults
		grep -l "SPC_ " | xargs sed -i 's/EY/CYS/'
		grep -l "SPC_ " | xargs sed -i 's/EFYO/CYS/'
		grep -l "BC_ "  | xargs sed -i 's/EFY/CYS/'
            else
		# Don't set missing restart file variables to defaults
		grep -l "SPC_ " | xargs sed -i 's/CYS/EFYO/'
		grep -l "SPC_ " | xargs sed -i 's/EY/EFYO/'
		grep -l "BC_ "  | xargs sed -i 's/CYS/EFY/'
            fi
```